### PR TITLE
.NET 6 update

### PR DESCRIPTION
**Pull request by ivanpaulovich**
.NET 6 update

### CODE REVIEW

**Comment by sergey-chanaev** [comment]
+1 for this. @yohanb @Zargath can anyone approve this pull request?

**Comment by ivanpaulovich** [comment]
Hey @AlexandreArpin,

Thanks for this useful tool. It seamless installs our certificates and I am enjoying it.

Could you take a look at this PR?

**Comment by Zargath** [comment]
Please don't pull people into PRs, we are all very busy and will get to this when we have time. In the mean time, feel free to pull the fork, compile it, and install the tool that way if lack of support for .NET 6 is a blocker for you. Thank you.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Setup .NET Coreq
```
Comment (yohanb) [inline .github/workflows/release.yml]: can you fix the typo while your at it please?
Thanks

**Comment by yohanb** [comment]
Simple review fix before I merge the PR.  As @Zargath said, please refrain from unnecessarily @mentioning someone to try to speed things up.  We all have day jobs and do our best to support our open-source contributions.
✌️

**Review by yohanb — APPROVED** [review]

**Comment by yohanb** [comment]
merged! Will release it today.  Stay tuned.

**Comment by ivanpaulovich** [comment]
Hey folks,

Sorry for the inconvenience. I wish you peace of mind to review  when you have time ;)

Best!

**Comment by sergey-chanaev** [comment]
Guys, appreciate for your support and apologise for being annoying!

**Pull request merged**